csharp/ql/src/Security Features/CWE-285/MissingAccessControl.qhelp

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Sensitive actions, such as editing or deleting content, or accessing admin pages, should have authentication checks 
+to ensure that they cannot be used by arbitrary users.
+</p>
+
+</overview>
+<recommendation>
+
+<p>
+Ensure that proper authorization checks are made for sensitive actions. 
+For WebForms applications, the <code>authorazation</code> tag in <code>Web.config</code> XML files 
+can be used to implement access control. The <code>System.Web.UI.Page.User</code> property can also be 
+used to verify a user's roles. 
+For MVC applications, the <code>Authorize</code> attribute can be used to require authorization on specific 
+action methods.
+</p>
+
+</recommendation>
+<example>
+
+<p>
+In the following WebForms example, the case marked BAD has no authorization checks; whereas the 
+case marked GOOD uses <code>User.IsInRole</code> to check for the user's role.
+</p>
+
+<sample src="WebForms.cs" />
+
+<p>
+The following <code>Web.config</code> file uses the <code>authorization</code> tag to deny access to anonymous users,
+in a <code>location</code>> tag to have it apply to a specific path.
```

Comment:
Typo + would it be feasible to update the language here. What does "it" refer to?
```suggestion
in a <code>location</code> tag to have it apply to a specific path.
```